fennel/client_tests/test_dataset.py

```diff
@@ -402,6 +404,11 @@ def pipeline_aggregate(cls, activity: Dataset):
                     into_field=str(cls.max_ratings),
                     default=0.0,
                 ),
+                Stddev(
```

Comment:
Can we add some tests in test_invalid_dataset.py where you try to break Stddev and ensures it breaks, like taking std dev of string field etc